lib/logger.py

```diff
@@ -87,7 +87,7 @@ def __init__(self, filename):
         data = self.f.read(12)
         self.start_time = datetime.datetime(*struct.unpack('HBBBBBI', data))
 
-    def read_gen(self, only_stream_id=None):
+    def read_gen(self, only_stream_id=None, multiple_streams=[]):
```

Comment:
I am not sure if it is worth to have both options. I would keep just one a checked if it is an iterable.